packages/slice-machine/src/apiClient.ts

```diff
@@ -79,6 +79,19 @@ export const createSlice = (
     .then((response: AxiosResponse<{ variationId: string }>) => response.data);
 };
 
+export const renameSlice = (
+  sliceId: string,
+  newSliceName: string,
+  libName: string
+): Promise<AxiosResponse> => {
+  const requestBody = {
+    sliceId,
+    newSliceName,
+    libName,
+  };
+  return axios.put(`/api/slices/rename`, requestBody, defaultAxiosConfig);
```

Comment:
you know axios can take a type parameter, not sure if it's useful here ?

Reply:
Is there an example of how this is used? :)

Reply:
`axios.put<MyType>(...).then(res => res.data // will be cast to MyType)`